src/components/Dropzone/Dropzone.tsx

```diff
@@ -210,7 +222,7 @@ const Dropzone = ({ className = '', onDrop, disabled = false, maxFiles, maxSize
                 className="sr-only"
                 onChange={onFileInputChange}
                 type="file"
-                accept={accept ? Object.entries(accept).map(([key]) => key).join(', ') : undefined}
+                accept={getAcceptedFileTypes(accept)}
```

Comment:
I converted this to a method because [browsers does not support](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types) image/heif as MIME type. So instead of use mimetype it uses the extensions .heic and .heif only for this type of files.